### PR TITLE
PVC: add redis and rabbitMQ PVC

### PR DESCRIPTION
**Pull request by jrcastro2**

### CODE REVIEW

**Comment by kpsherva** [comment]
silly question: could you rename the commit to
`deployments: add redis and rabbitmq persistent volume claims` 
? I am sure it will be easier to search in the history than the acronym :)

**Review by kpsherva — APPROVED** [review]

File: invenio/values.yaml [hunk 1]
```diff
@@ -127,6 +127,20 @@ persistence:
   size: 10G
   storage_class: ""
 
+mq_persistence:
```
Comment (ntarocco) [inline invenio/values.yaml:130]: I would propose to move these and the redis one inside related keys, e.g.:
```
redis:
    ...
    persistency:
        enabled: 
        ...

File: invenio/values.yaml [hunk 2]
```diff
@@ -127,6 +127,20 @@ persistence:
...
+  enabled: true
+  name: "rabbitmq-volume"
+  access_mode: ReadWriteMany
+  size: 10G
+  storage_class: "cephfs-ssd-no-backup"
```
Comment (ntarocco) [inline invenio/values.yaml:135]: This is a CERN value. As discussed, maybe make it `""`? (it should take the default)
We need then to override this in our CERN configs and use this value there.

File: invenio/values.yaml [hunk 3]
```diff
@@ -139,6 +146,12 @@ rabbitmq:
   secret_name: "mq-secrets"
   default_password: "mq_password"
   celery_broker_uri: "amqp://guest:mq_password@mq:5672/"
+  persistence:
+    enabled: true
+    name: "rabbitmq-volume"
+    access_mode: ReadWriteMany
+    size: 10G
+    storage_class: ""
```
Comment (kpsherva) [inline invenio/values.yaml:154]: whats the consequence of the empty string as a storage_class? is it an optional param?

File: invenio/values.yaml [hunk 4]
```diff
@@ -131,6 +131,13 @@ redis:
   enabled: true
   image: "redis:7"
   host: ""
+  persistence:
+    enabled: true
+    name: "redis-volume"
+    access_mode: ReadWriteMany
```
Comment (ntarocco) [inline invenio/values.yaml:137]: Minor: it is probably enough `ReadWriteOnce`, given that it is not shared. Anyway, we can keep it like this.

**Review by ntarocco — APPROVED** [review]

**Pull request merged**